### PR TITLE
fix logic in Auditorium (v1) where row/column 0 is treated as false instead of being a valid seat

### LINKED ISSUE #1422: cannot take a seat in "0-th" row in the auditorium
cannot-take-seat-middlerow

you can see that no seat assigned when I click in that "middle row"

### PR DESCRIPTION
**Pull request by yarikoptic**
There is a separate check for FireLane (column) so should be ok in that regard
now that it would return True for 0, 0

Closes #1422

### CODE REVIEW

**Review by 0xdevalias — CHANGES_REQUESTED** [review]
Nice discovery @yarikoptic!

A couple of little improvements then this should be good to go!

File: src/components/templates/Audience/Audience.tsx [hunk 1]
```diff
@@ -229,8 +229,7 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
 
     return recentVenueUsers.filter((user) => {
       const { row, column } = user.data?.[venueId] ?? {};
-
-      return row && column;
+      return row != null && column != null;
```
[the remark this answers is not in the document]
Comment (yarikoptic) [reply src/components/templates/Audience/Audience.tsx]: Thanks! Applied suggestion. Will do the rest when get to laptop. Feel welcome to push such changes straight to any of my PRs in the future - they should be open for contributors

[the remark this answers is not in the document]
Comment (0xdevalias) [reply src/components/templates/Audience/Audience.tsx]: > Feel welcome to push such changes straight to any of my PRs in the future - they should be open for contributors

Sounds good, shall keep it in mind.

For context, for similar reasons as you needing to get on a computer for it, it takes extra effort to switch context/clone your branch/commit the changes/push them back than it does to detail them as part of a review. Which is why I tend to write them here first.

[the remark this answers is not in the document]
Comment (yarikoptic) [reply src/components/templates/Audience/Audience.tsx]: Yeap, but between two of us causes duplication + delay. Anyways, that was just my FWIW and just me reflecting on how I approach PRs where I see that there is substantially more to be done outside of the diff at hand (although I did get burnt with such strategy too ;-)).

[the remark this answers is not in the document]
Comment (0xdevalias) [reply src/components/templates/Audience/Audience.tsx]: Haha, yeah, that's fair enough :)

[the remark this answers is not in the document]
Comment (yarikoptic) [reply src/components/templates/Audience/Audience.tsx]: pushed

[the remark this answers is not in the document]
Comment (0xdevalias) [reply src/components/templates/Audience/Audience.tsx]: Cool, thanks!

**Review by 0xdevalias — APPROVED** [review]
Code changes look good here, if we've clicked through this and are happy that it works as expected still, then LGTM!

**Edit:** Been fighting with our deploy preview environment script to try and get it working for non-sparkle PRs, which is why I haven't merged this yet.  Will probably end up cloning and testing This PR locally since it's taking so long, then try and fix that GitHub action/script more later.

**Comment by 0xdevalias** [comment]
@mike-lvov Thanks for checking + merging this.

@yarikoptic Thanks for the bug fix!

**Pull request merged**